WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -84,6 +77,24 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
     return [super hitTest:point withEvent:event];
 }
 
+- (void)didMoveToSuperview
+{
+    [super didMoveToSuperview];
+    if (self.didPreserveStartingConstraintConstants) {
```

Comment:
Mmmmmm after some debugging, i think it's safe, currently. Since `_didPreserve` is true when the superview is nil.

False alarm i believe!